ui/packages/consul-ui/mock-api/v1/internal/ui/service-topology/_

```diff
@@ -53,17 +53,11 @@ ${
     fake.seed(index);
 
 
-    // Randomly make permissive intentions
-    const defaultAllow = fake.random.boolean();
```

Comment:
I thought about keeping this, but we verified with BE that default allow and wildcard intentions can co-exist.